src/05-invoicing/invoices.apib

```diff
@@ -141,6 +141,7 @@ Get details for a single invoice.
                                         + excluding
                             + tax (object)
                                 + rate: 0.21 (number)
```

Comment:
Q: since we are now exposing the Tax Rate ID, should we remove this and only send the ID?

Reply:
I vote on only sending the UUID like in the draft tax_rate_id

Reply:
I think it would make more sense indeed, if we replace tax with taxRate and only provide the id.

```
+ tax_rate (object)
    + type: `tax_rate` (string)
    + id: `4b0f959a-27e3-4080-b1b2-a05b374863cb` (string)
```

On the other hand, there currently is no `taxRates.info` endpoint. So if you want to know the rate or the name, you need to fetch them via the `taxRates.list` endpoint, which is a bit weird.

Reply:
The taxRates.list doesn't allow to filter on the taxRate id. @PieterReel should we then add a taxRates.info or a an if filter on the list?

Reply:
@luisabritotl IMO both options are valid.

@lowiebenoot why do you need the `type`? and why not just `+ tax_rate_id: `4b0f959a-27e3-4080-b1b2-a05b374863cb` (string)`

Reply:
I think Daniel that is a matter of consistency

Reply:
Indeed,  I just realized that we add a `type` for almost "all" objects.